common/src/main/java/org/opensearch/ml/common/connector/AbstractConnector.java

```diff
@@ -44,6 +44,7 @@ public abstract class AbstractConnector implements Connector {
     public static final String OWNER_FIELD = "owner";
     public static final String ACCESS_FIELD = "access";
     public static final String CLIENT_CONFIG_FIELD = "client_config";
+    public static final String MODEL_INTERFACE_FIELD = "model_interface";
```

Comment:
Move to model layer and change field name to "interface"?

Reply:
Ack